sensu/plugins/check-l3-router.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOSTNAME=`hostname`
+for IFACE in $(ifquery --list); do
```

Comment:
Specifically checking eth0/eth0.400 will not handle all cases when networking may be split across multiple NICs as is the case in SoftLayer and possibly others. This would also fail if we ever used a non-400 VLAN for the ucarp interface.
But, not sure on a better alternative to handle non-eth0 cases.